src/Psl/Result/tryResultFrom.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Result;
+
+use Exception;
```

Comment:
Would it make sense to use `Throwable` instead?

Reply:
no, as we are trying to wrap userland exception, Errors are usually thrown by PHP and they should not be expected ( e.g TypeError )